man/systemd.network.xml

```diff
@@ -1056,6 +1056,25 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>InitialCongestionWindow=</varname></term>
+          <listitem>
+            <para>The TCP initial congestion window or <literal>InitialCongestionWindow</literal>  is used during the start
+            of a TCP connection. During the start of a TCP session, when a client requests for a resource, the servers initcwnd
```

Comment:
"servers" needs an apostrophe: "the server's initcwnd"

I would also avoid introducing the "initcwnd" abbreviation here. Just spell out "initial congestion window" again.